.github/workflows/integration-tests.yaml

```diff
@@ -2,9 +2,9 @@ name: Integration Tests
 
 on:
   push:
-    branches: [ master ]
+    branches: [ main ]
```

Comment:
KInda dislike "politically correct" branch naming... 😞

Reply:
This change has nothing to do with p.correctness, and everything with consistency... I looked at few random quarkiverse extensions, and quarkus itself, and all use `main`, so I assume some Ecosystem CI jobs by default target `main` branch.